src/components/project-list.jsx

```diff
@@ -1,21 +1,43 @@
+import { graphql, Link, useStaticQuery } from 'gatsby'
+import Img from 'gatsby-image'
 import React from 'react'
 import styled from 'styled-components'
 import media from 'styled-media-query'
-import { Link } from 'gatsby'
-import PageContainer from './page-container'
 import background from '../assets/howly-bg.svg'
-import logo from '../assets/images/howly-logo.png'
+import PageContainer from './page-container'
 
-const Project = props => (
-  <ProjectContainer to="/howly">
-    <img src={logo} alt="Howly logo" />
-    <h2>
-      React Native, Node.js, AWS,
-      <br /> Serverless, Product Design
-    </h2>
-    <p>The snapchat of events, Howly keeps you and your friends connected</p>
-  </ProjectContainer>
-)
+const Project = () => {
+  const data = useStaticQuery(graphql`
```

Comment:
Not sure if this should be within the component function or not?

Reply:
Yeah, I had it outside, but most of the examples were using the useStaticQuery, so went with that approach, Can change if you think having it outside is better.

Reply:
Roger that, I’m easy... I guess whatever is more performant although would be nice to use the same pattern throughout.